src/Repository/Project/PhaseRepository.php

```diff
@@ -24,4 +24,35 @@ public function getByEtudeQuery(Etude $etude)
 
         return $qb;
     }
+
+    /**
+     * @return \Doctrine\ORM\QueryBuilder
+     */
+    public function getByEtudeQueryPvId(int $pv_id, Etude $etude)
```

Comment:
Je viens de relire le code du procesVerbalSubType, j'ai du plonger dans l'implémentation pour comprendre ce que faisait cette fonction. Ce qui veut dire que l'on peut probablement améliorer le naming des 2 fonctions.

```suggestion
    public function getByEtudeAndPv(ProcesVerbal $pv, Etude $etude)
```

Reply:
Ici je passe le pv_id en argument, pas le Proces verbal, je fais quand même le changement ?

Reply:
Oui, c'est mieux. Ca rend l'interface de la fonction plus lisible: vu de l'appellant, un argument de type int n'apporte pas d'information: on pourrait y mettre `rand(0,100)` et ca marcherait quand même.

Là on force l'appellant à passer un ProcesVerbal.

+ dans la suite de l'autre commentaire (celui sur combiner les 2 fonctions de query en une), tu pourrais finalement faire

```php
public function getByEtudeAndPv(Etude $etude, ProcesVerbal $pv = null)
```

et comme ça à l'intérieur de la fonction, tu peux adapter la condition SQL selon que $pv est null ou pas.